workflow/artifacts/git/git.go

```diff
@@ -115,9 +116,15 @@ func (g *ArtifactDriver) Load(inputArtifact *wfv1.Artifact, path string) error {
 		return err
 	}
 	defer closer()
+
+	var recurseSubmodules = git.DefaultSubmoduleRecursionDepth
+	if inputArtifact.Git.DisableSubmodules {
+		log.Info("Setting recurse submodules to NoRecurseSubmodules")
```

Comment:
Perhaps more meaningful/readable log here, e.g. "Recursive cloning of submodules is disabled".

Reply:
Done